packages/psss/src/routes/PrivateRoute.js

```diff
@@ -6,43 +6,67 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Redirect, Route } from 'react-router-dom';
-import { checkIsLoggedIn } from '../store';
+import { checkIsLoggedIn, getCurrentUser } from '../store';
 
-/*
- * A wrapper for <Route> that redirects to the login
- * screen if you're not yet authenticated.
- * */
-export const PrivateRouteComponent = ({ isLoggedIn, children, ...props }) => {
-  return (
-    <Route
-      {...props}
-      render={({ location }) => {
-        return isLoggedIn ? (
-          children
-        ) : (
+export const PrivateRouteComponent = ({
+  isLoggedIn,
+  authCheck,
+  currentUser,
+  children,
+  redirectTo,
```

Comment:
Even though it's kinda unnecessary now, I really like that `redirectTo` can be overridden!